outputs/alertmanager.go

```diff
@@ -36,6 +37,9 @@ func newAlertmanagerPayload(falcopayload types.FalcoPayload) []alertmanagerPaylo
 	amPayload.Annotations["info"] = falcopayload.Output
 	amPayload.Annotations["summary"] = falcopayload.Rule
 
+	// Prefix alertname
+	amPayload.Labels["alertname"] = "Falco" + strcase.ToCamel(string(falcopayload.Rule))
```

Comment:
Is that mandatory to add `Falco` as prefix?

I think the label should be `rule` instead of `alertname` wdyt?

Reply:
@nibalizer 
@epcim

wdyt?

Reply:
will check how do others, on our impl. we use alertname. 
btw we already have summary label with Rule content, this was mostly to really create "alertname" for Alertmanager

Reply:
Let's ask the local prometheus expert @leogr what they think?

Reply:
If the idea is just to tag these as coming from falco somehow, the sidekick supports adding custom k,v pairs in the config file.